src/simple_job/test_load_books.py

```diff
@@ -1,18 +1,25 @@
 from datetime import date
 from pyspark.sql import DataFrame
 from awsglue.context import GlueContext
-from pyspark.sql import DataFrame
-from awsglue.context import GlueContext
 from unittest.mock import patch, Mock, call, ANY
 from glue_biscuit.test import (
     DataFrameMatcher,
     DynamicFrameMatcher,
     spark_context,
     mock_glue_context,
 )
+from glue_biscuit.schema_utils import schema_from_cloudformation
+import pytest
 from simple_job.load_books import main, load_books, save_books
 
 
+@pytest.fixture(autouse=True)
+def mock_schema_from_glue():
+    with patch("simple_job.load_books.schema_from_glue") as mock_sink_func:
+        mock_sink_func.side_effect = lambda *args: schema_from_cloudformation('./template.yml', args[1])
+        yield mock_sink_func
+
+
```

Comment:
This worked out rather nicely after adding a requirement for the glue context to be passed in. The end user only had to end up mocking their own schema func.

Reply:
(or, rather, the things they used inside it)